courses/level101/big_data/evolution.md

```diff
@@ -45,41 +45,42 @@
 # Other tooling around Hadoop
 
 1. [**Hive**](https://hive.apache.org/)
-    1. Uses a language called HQL which is very SQL like.  Gives non-programmers the ability to query and analyze data in Hadoop.  Is basically an abstraction layer on top of map-reduce.
+    1. Uses a language called HQL which is very SQL like. Gives non-programmers the ability to query and analyze data in Hadoop. Is basically an abstraction layer on top of map-reduce.
     2. Ex. HQL query:
-        1. _SELECT pet.name, comment FROM pet JOIN event ON  (pet.name = event.name);_
+        1. `SELECT pet.name, comment FROM pet JOIN event ON (pet.name = event.name);`
     3. In mysql:
-        1. _SELECT pet.name, comment FROM pet, event WHERE  pet.name = event.name;_
+        1. `SELECT pet.name, comment FROM pet, event WHERE pet.name = event.name;`
 2. [**Pig**](https://pig.apache.org/)
-    1. Uses a scripting language called Pig Latin, which is more workflow driven.  Don't need to be an expert Java programmer but need a few coding skills.  Is also an abstraction layer on top of map-reduce.
+    1. Uses a scripting language called Pig Latin, which is more workflow driven. Don't need to be an expert Java programmer but need a few coding skills. Is also an abstraction layer on top of map-reduce.
     2. Here is a quick question for you:
-    What is the output of running the pig queries in the right column against the data present in the left column in the below image?
+    What is the output of running the Pig queries in the right column against the data present in the left column in the below image?
 
     ![Pig Example](images/pig_example.png)
 
     Output:
-    ```
+    <pre><code>
```

Comment:
does the backquote not work here? why introduce html tags?

Reply:
Backquotes (```) in lists render the code blocks inline.  (The below snap is from [https://linkedin.github.io/school-of-sre/level101/big_data/evolution/](https://linkedin.github.io/school-of-sre/level101/big_data/evolution/))

![backquote](https://github.com/linkedin/school-of-sre/assets/94439978/b1893369-d0f0-45d7-8835-be23822cea77)